redis/db/redisbp/monitored_client.go

```diff
@@ -7,9 +7,52 @@ import (
 	"time"
 
 	"github.com/go-redis/redis/v8"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
 	"github.com/reddit/baseplate.go/metricsbp"
 )
 
+const (
+	nameLabel  = "name"
+	stateLabel = "state"
+
+	stateTotal = "total"
+	stateIdle  = "idle"
+	stateStale = "stale"
+)
+
+var (
+	gaugeLabels = []string{
+		nameLabel,
+	}
+
+	hitsGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
these seem like all the same metric

```
redisbp_pool_gets_total{name="$name",result="hit|miss|timeout"}
```

This lets you get an overall QPS while still breaking it down

https://prometheus.io/docs/practices/naming/#labels